recipes/libsafec/all/conanfile.py

```diff
@@ -99,6 +104,14 @@ def generate(self):
             f"--enable-strmax={self.options.strmax}",
             f"--enable-memmax={self.options.memmax}",
         ]
+
+        if is_apple_silicon:
```

Comment:
`x86_64` arch also needs those flags:

```suggestion
        if is_apple_os(self) and self.settings.arch in ["x86_64", "armv8"]:
```

Reply:
Sure, just pushed the commit, assuming the condition should be inlined and add x86_64 to the affected archs
The PR description will be updated accordingly